DESCRIPTION

```diff
@@ -19,8 +19,13 @@ Authors@R: c(
     person("Junlue", "Zhao", role = "aut"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
   )
-Description: Filter module for teal applications.
-License: Apache License 2.0 | file LICENSE
+Description: 
+    Module containing filter panel for 'teal' applications. Allows for interactive filtering of data stored in 
+    'data.frame' and 'MultiAssayExperiment' objects. Also displays filtered and unfiltered observation counts.
```

Comment:
```suggestion
A filter panel module for 'teal' applications that allows interactive and reproducible data filtering.
```

I want to be careful to mention MAE in teal.slice. We have some discussion on moving this extension away from the core package.
Also, I want to mention about the reproducible part.
Let me know what you think.

Reply:
It does handle MAE now so it should be mentioned, shouldn't it? When MAEs are moved to a different package, vignettes will be updated (in more places than one).

```
Module containing filter panel for 'teal' applications. Allows for interactive filtering of data stored in 'data.frame' and 'MultiAssayExperiment' objects nad returns reproducible code of the filtering operations. Also displays filtered and unfiltered observation counts.
```